bin/monitor_queue_sizes

```diff
@@ -0,0 +1,8 @@
+#/usr/bin/env bash
+
+# Once a minute, emit the number of jobs in each queue to Librato
+
+while true; do
+  bundle exec rake monitor:queue_sizes
+  sleep 60
```

Comment:
Since it's easy enough to do, I set it up with a default in @bf2414c.  It's not really a timeout though, it's more of a "wait between attempts" -- how do you feel about the var name?